libbpf-cargo/src/btf/btf.rs

```diff
@@ -496,6 +498,16 @@ impl<'a> Btf<'a> {
                         writeln!(def, r#"        }}"#)?;
                         writeln!(def, r#"    }}"#)?;
                         writeln!(def, r#"}}"#)?;
+                    } else if !t.is_struct {
+                        // write a Debug implementation for a union
+                        writeln!(def, r#"impl std::fmt::Debug for {} {{"#, t.name)?;
+                        writeln!(
+                            def,
+                            r#"    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {{"#
+                        )?;
+                        writeln!(def, r#"        write!(f, "unimplemented debug for union")"#)?;
```

Comment:
Maybe this, to keep it more terse? 
```suggestion
                        writeln!(def, r#"        write!(f, "(???)"#)?;
```

Reply:
+1